src/tooling/piral-cli/src/injectors/pilet.ts

```diff
@@ -52,6 +64,48 @@ export default class PiletInjector implements KrasInjector {
     );
   }
 
+  loadRemoteFeed(feed: string): Promise<PiletMetaData[]> {
+    return new Promise((resolve) => {
+
+      if (!/^https?:/.test(feed)) {
+        log('generalWarning_0001', 'Feed must be an absolute URL.');
+        resolve();
+      }
+
+      const h = feed.startsWith('https') ? https : http;
```

Comment:
To simplify things you could use `axios`. We already leverage it in other modules on the `piral-cli`, so its not a new dependency (and a lot more convenient).

Reply:
ok, nice
didn't want to introduce another dependency just because of that - and also somehow missed that `axios` is already there - will make use of it 👍

Reply:
Great - regarding use of `Axios`: We embed this in the `piral-cli` to avoid resolving dependencies on install (i.e., to boost things like `npm init piral-instance`). To use it you need to import `axios` from the `externals` module (https://github.com/smapiot/piral/blob/develop/src/tooling/piral-cli/src/external/index.ts).

Reply:
👍 VS Code automatically found the right place. Guess I could have tried that earlier already 😊